src/Common/tests/System.Net/Sockets/SocketTestExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.Sockets.Tests
+{
+    static class SocketTestExtensions
+    {
+        // Binds to an IP address and OS-assigned port. Returns the chosen port.
+        public static int Bind(this Socket socket, IPAddress address)
```

Comment:
Looks like an overload for Bind. In my opinion this is confusing. Maybe using a simple static helper function would be easier to understand.